test/apps/mapbox-layers/package.json

```diff
@@ -5,9 +5,10 @@
   },
   "dependencies": {
     "deck.gl": "^8.4.0",
-    "react": "^16.3.0",
-    "react-dom": "^16.3.0",
-    "react-map-gl": "^5.0.0"
+    "mapbox-gl": "^2.0.0",
```

Comment:
Should we also have a maplibre example? The packages are diverging overtime and it's a lot of effort to switch this example back and forth between them at the moment in testing.

Reply:
Yes I agree. I will open another PR to consolidate test apps.